### ISSUE TITLE
lesson_initialize.py fails to initialize new lessons

### ISSUE THREAD

**Issue by annefou** [carpentries/styles#336]
I probably misunderstood something but I tried to follow instructions given at https://carpentries.github.io/lesson-example/setup.html but it failed to initialize the lesson:

```
$ python bin/lesson_initialize.py
Warning: CODE_OF_CONDUCT.md already exists.
**Exiting without creating files.**
```
So I deleted `CODE_OF_CONDUCT.md` and then it worked:

```
$ rm CODE_OF_CONDUCT.md
$ python bin/lesson_initialize.py
```

**Comment by maxim-belkin** [comment]
I'll look into this.

**Comment by maxim-belkin** [comment]
You're right, Anne. Francois added `CODE_OF_CONDUCT.md` to the root directory in 4755a6d in an attempt to fix carpentries/styles#300. This whole situation is caused by the approach we're currently employing in that lesson use this repository as their base.

Currently, the solution would be to fix the `lesson_initialize.py` script to handle existence of `CODE_OF_CONDUCT.md` file.

**Comment by rgaiacs** [comment]
Thanks for the issue. Fixed by https://github.com/carpentries/styles/pull/338.
